test/local-mode.ts

```diff
@@ -1,6 +1,7 @@
 import * as assert from 'assert';
 import { expect } from 'chai';
 import * as Docker from 'dockerode';
+import * as fs from 'fs';
```

Comment:
This is unused, and for future reference, we tend to use `import { fs } from 'mz';` to get the free promisified functions.